### PR TITLE
Bucks beta ps

### PR DESCRIPTION
**Pull request by GraemeMcCubbin**
- Add the Bucks CC Beta project story to production
- Replace the Bucks CC Discovery and Alpha project story tile with the Bucks CC Beta project story tile on the homepage
- Add the Bucks CC Beta project story tile to the Project Stories index page

### CODE REVIEW

File: source/project-stories/bucks-cc-beta.erb [hunk 1]
```diff
@@ -0,0 +1,210 @@
+---
+title: "Project Story - Buckinghamshire County Council"
+
+header:
+  client: "Buckinghamshire County Council"
+  introduction: "Digitalising key local government services with Buckinghamshire County Council: The Beta phase"
+  image: "/project-stories/bucks-cc-beta/header.jpg"
+  image_description: ""
```
Comment (mybits) [inline source/project-stories/bucks-cc-beta.erb]: Let's put some content in here, otherwise will be missing img `alt`

**Comment by mybits** [comment]
I left one comment re img `alt` attribute missing.
There would be also a content cutting here I'm afraid :) :
screen shot 2016-09-26 at 11 45 00

And also header image looks a bit blurry on retina. It would be good to change it to higher res if possible.

Otherwise looks good 🌴

File: source/project-stories/bucks-cc-beta.erb (hunk 1 above)
Comment (GraemeMcCubbin) [reply source/project-stories/bucks-cc-beta.erb]: Hey @mybits - thanks for pointing this out, have added in an image description for the header.

**Comment by GraemeMcCubbin** [comment]
Thanks @mybits - have trimmed the text under that section. Will update the image header at a later time.

**Pull request closed without merging**